editor/script_create_dialog.cpp

```diff
@@ -582,6 +582,9 @@ void ScriptCreateDialog::_bind_methods() {
 	ClassDB::bind_method("_path_changed", &ScriptCreateDialog::_path_changed);
 	ClassDB::bind_method("_path_entered", &ScriptCreateDialog::_path_entered);
 	ClassDB::bind_method("_template_changed", &ScriptCreateDialog::_template_changed);
+
+	ClassDB::bind_method("config", &ScriptCreateDialog::config);
```

Comment:
If you want the arguments named in the doc, you need to name them here.

Reply:
Ahhh, thank you. Didn't catch this.